Project-Starlord/Controllers/AccountController.cs

```diff
@@ -147,6 +147,21 @@ public async Task<ActionResult<UserModel>> Register(UserModel userModel)
             return userModel;
         }
 
+        [Authorize]
+        [HttpGet("{query}")]
```

Comment:
Why is it called query the variable? I think filterValue would be a better name or something like that, But we can discuss this together.

Reply:
we can even call it `input` or indeed `filterValue`

Reply:
Can you call it filterValue? After that the PR is all good.

Reply:
done